packages/pipeline/src/parsers/radar_orders/index.ts

```diff
@@ -0,0 +1,91 @@
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import { RadarBook, RadarMarket, RadarOrderType, RadarSignedOrder } from '@radarrelay/types';
+import * as R from 'ramda';
+
+import { aggregateOrders, GenericRawOrder } from '../utils';
+
+import { TokenOrderbookSnapshot as TokenOrder } from '../../entities';
+import { OrderType } from '../../types';
+
+/**
+ * Marque function of this file.
+ * 1) Takes in orders from an orderbook,
+ * other information attached.
+ * @param radarOrderbook A raw orderbook that we pull from the radar API.
+ * @param radarMarket An object containing market data also directly from the API.
+ * @param observedTimestamp Time at which the orders for the market were pulled.
+ * @param source The exchange where these orders are placed. In this case 'radar'.
+ */
+export function parseRadarOrders(
+    radarOrderbook: RadarBook,
+    radarMarket: RadarMarket,
+    observedTimestamp: number,
+    source: string,
+): TokenOrder[] {
+    const transformToGeneric = (radarOrder: RadarSignedOrder) => _toGeneric(radarMarket, radarOrder);
```

Comment:
can we call `_toGeneric` directly instead of using this? `radarOrderBook.bids.map(order => _toGeneric(radarMarket, order))`